src/docs/development/tools/sdk/overview.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Flutter SDK overview
+short-title: Flutter SDK
+description: Flutter libraries and command-line tools.
+---
+
+The Flutter SDK has the packages and command-line tools that you need to develop
+Flutter apps across platforms. To get the Flutter SDK, see [Install][].
+
+## What's in the Flutter SDK
+
+The following is available through the Flutter SDK:
+
+* [Dart SDK][]
+* Heavily optimized, mobile-first 2D rendering engine with
+  excellent support for text
+* Modern react-style framework
+* Rich set of widgets implementing Material Design and iOS styles
+* APIs for unit and integration tests
+* Interop and plugin APIs to connect to the system and 3rd-party SDKs
+* Headless test runner for running tests on Windows, Linux, and Mac
+* [Dart DevTools][] for testing, debugging, and profiling your app
+* `flutter` and `dart` command-line tools for creating, building, testing,
+  and compiling your apps
+
+Note: For more information about the Dart SDK, see its
+[README file][].
+
+## `flutter` command-line tool
+
+The `flutter` CLI is how developers (or IDEs on behalf of developers) interact with Flutter.
+For more information about `flutter`, see the [`flutter` reference page][].
+
+## `dart` command-line tool
+
+The `dart` CLI tool is available with the Flutter SDK. To use the `dart` CLI included with
+the Flutter SDK, add the following to your `PATH`:
+
+```terminal
+$PATH:~/flutter/bin/dart
```

Comment:
The PATH location is going to be user-dependent. If they've followed the "update your path" advice from the install pages, it should just work. So I wouldn't call this out. The path needs to be updated in both cases (for flutter and dart).

Also, @kwalrath, is https://dart.dev/tools/dart-vm the best page to link to for dart CLI info? It looks like the dart.dev pages haven't been updated yet and still use dartfmt, dartanalyzer, etc.

Reply:
how about this page? https://dart.dev/tools/dart-tool

Reply:
> how about this page? https://dart.dev/tools/dart-tool

Yes, that's the page to use.